app/src/main/res/values/strings.xml

```diff
@@ -332,4 +332,9 @@
     <string name="show_archived_cards">Show archived cards</string>
     <string name="view_online">View online</string>
     <string name="field_may_not_be_empty">Field may not be empty</string>
+    <string name="action_more_options">More options</string>
+    <string name="addWithoutBarcode">Add a card without barcode</string>
+    <string name="enter_card_id">Enter the ID number or text on your card</string>
+    <string name="card_id_may_not_be_empty">Card ID may not be empty</string>
```

Comment:
I dislike "may not be"; "should not be" or "must not be" is less ambiguous, which is also easier for translators.

Also applies to `field_may_not_be_empty` I guess.